### PR TITLE
Enable to use postgres connector from CLI

### PR DESCRIPTION
**Pull request by olafmaurer**
Also fixes a number of bugs in the postgres connector. Finally, it adds a text file with sql commands that can be used to create an example table for the postgres connector.

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -111,6 +114,8 @@ func GetStorageReader(config Config) (storageconnector.StorageReader, error) {
 		return storageconnector.NewMemoryStorage(data), nil // implement a generator of random list of features?
 	case StorageTypeGit:
 		return storageconnector.NewGitStorage(config.Git.URL)
+	case StorageTypePG:
+		return storageconnector.NewPostgresStorage("omaurer", "pazuzu") // FIXME: this is for testing, it needs to be properly put in the config file
```
Comment (netme) [inline config.go]: That's will be great for presentation, but can we make it configurable?

File: storageconnector/pgconnector.go [hunk 2]
```diff
@@ -30,39 +37,46 @@ func (store *postgreStorage) connect() error {
...
 	var snippet string
 	var index int
-	store.connect()
-	defer store.disconnect()
+	err := store.connect()
+	if err != nil {
+		return nil, err
+	}
+	//fmt.Println("Sql query:")
+	//fmt.Println(SqlQuery)
```
Comment (netme) [inline storageconnector/pgconnector.go]: It's better not to commit commented code.

File: storageconnector/pgtestsql.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# simple sql querys to create a demo table for pazuzu in postgres
```
Comment (netme) [inline storageconnector/pgtestsql.txt]: 2 things:
- We can already change extensions to `.sql`
- We have a folder called `storageconnector/fixtures` which fits much better for such files

My suggestion will be to call it something like: `storageconnector/fixtures/fixtures.sql`

File: storageconnector/pgtestsql.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+CREATE TABLE IF NOT EXISTS features (index int, name text, description text, author text, lastupdate timestamptz, dependencies text, snippet text);
+INSERT INTO features VALUES (1, 'java', 'openjdk8', 'Olaf', '2016-11-25 11:36:25+01', '', 'sudo apt install openjdk-8-jre');
+INSERT INTO features VALUES (2, 'python2', 'python2', 'Olaf', '2016-11-25 11:41:20+01', '', 'sudo apt install python');
+INSERT INTO features VALUES (3, 'python3', 'python3', 'Olaf', '2016-11-25 11:41:20+01', '', 'sudo apt install python3');
+INSERT INTO features VALUES (4, 'numpy', 'python2 package numpy', 'Olaf', '2016-11-25 11:43:50+01', 'python2', 'sudo apt install python-numpy');
+INSERT INTO features VALUES (5, 'numpy3', 'python3 package numpy', 'Olaf', '2016-11-25 11:44:40+01', 'python3', 'sudo apt install python3-numpy');
+INSERT INTO features VALUES (6, 'numpy-all', 'python2+python3 package numpy','Olaf', '2016-11-25 11:45:30+01    ', 'python2 python3', 'sudo apt install python-numpy && sudo apt install python3-numpy');
+
```
Comment (netme) [inline storageconnector/pgtestsql.txt]: Examples are awesome. Suggestion for the future - add complex examples, e.g. `A-python-with-numpy`. Ignore it for now.

**Comment by rdllopes** [comment]
👍

File: storageconnector/pgconnector.go (hunk 2 above)
Comment (olafmaurer) [inline storageconnector/pgconnector.go]: Ok, I removed it.

File: storageconnector/pgtestsql.txt (hunk 4 above)
Comment (olafmaurer) [inline storageconnector/pgtestsql.txt]: Yeah. Good idea.

File: config.go (hunk 1 above)
Comment (olafmaurer) [inline config.go]: Yes we should and we should discuss where to put it. Probably somewhere in .pazuzu-cli.yaml

Comment (olafmaurer) [inline config.go]: I will go ahead with merge now, this requested change is now Issue #117.

**Comment by netme** [comment]
👍

**Pull request merged**